vector-app/src/androidTest/java/im/vector/app/CantVerifyTest.kt

```diff
@@ -76,4 +81,19 @@ class CantVerifyTest {
         Espresso.onView(ViewMatchers.withText(R.string.bottom_sheet_setup_secure_backup_title))
                 .check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
     }
+
+    @After
+    fun tearDown() {
```

Comment:
The existing ClearCurrentSessionRule was not working as expected?

Reply:
Actually it works when the test is started, but application opens up way before test is started by runner. And in our app `onCreate`, we are trying to get and initialize active session. That throws error if you executed an instrumented test last time but didn't clear that session, possibly because of being in ambiguous state.